SU-HSDO/suhumsci
# HSD8-704 Carousel paragraph type

# READY FOR REVIEW

# Summary
- Created a carousel paragraph type

# Need Review By (Date)
- 9/25

# Urgency
- medium

# Steps to Test
1. checkout this branch
1. `composer install --prefer-source`
1. either `drush @[site].local cim -y` or `drush @[site].local si config_installer -y`
1. create a basic page
1. create a carousel in the hero area and in the main components area
1. ensure it functions lie you'd expect a carousel to function

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
